Cargo.toml

```diff
@@ -105,6 +105,7 @@ debug = 0
 strip = true
 
 [patch.crates-io]
+libc = { git = "https://github.com/thesummer/libc", rev = "2ada946801aec517051bb00cb83e890f9af5ad37" }
```

Comment:
This commit is only for convenience in case someone wants to build locally.  I will remove this before merging.